dubbo-configcenter/dubbo-configcenter-zookeeper/src/main/java/org/apache/dubbo/configcenter/support/zookeeper/ZookeeperDynamicConfigurationFactory.java

```diff
@@ -28,6 +29,12 @@
 
     private ZookeeperTransporter zookeeperTransporter;
 
+    {
+        ZookeeperTransporter extension = ZookeeperTransporter.getExtension();
```

Comment:
It's better to put it into the constructor.

Reply:
fine.